### PR TITLE
Add JunLang

### PR DESCRIPTION
**Pull request by lemonorangeapple**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").
- [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is `JunLang`

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by lemonorangeapple** [comment]
@packagecontrol-bot .

File: channel.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
 		"https://raw.githubusercontent.com/keeganstreet/sublime-specificity/master/packages.json",
 		"https://raw.githubusercontent.com/kik0220/sublimetext_japanize/master/packages.json",
 		"https://raw.githubusercontent.com/kylederkacz/lettuce-farmer/master/packages.json",
+		"https://raw.githubusercontent.com/Jun-Software/JunLang_package/package/package.json",
```
Comment (braver) [inline channel.json:71]: Please remove this line, it's not needed. Just your repo listed in j.json is enough.

File: repository/j.json [hunk 2]
```diff
@@ -1404,6 +1404,17 @@
 				}
 			]
 		},
+		{
+			"name": "JunLang", 
+			"details": "https://github.com/Jun-Software/JunLang_package",
+			"labels": ["JunLang"],
```
Comment (braver) [inline repository/j.json:1410]:
```suggestion
			"labels": ["JunLang", "language syntax"],
```

**Review by braver — CHANGES_REQUESTED** [review]

**Comment by braver** [comment]
Please have a look the [scope naming guide](https://www.sublimetext.com/docs/scope_naming.html) and ensure your scopes all end with `.junlang`.

**Comment by lemonorangeapple** [comment]
@braver I changed them. Can it work now?

**Comment by braver** [comment]
That's better :) Please do add a README, otherwise people won't really understand what your package is about. For instance, link to information about the language your package supports. 

Also please don't forget to tag a new release with your improvements, package control only sees your tags: other changes are just sitting there in your repo and won't make it to end users. 

Let's not let that stop us from shipping this. Just do those things in your own time.

**Pull request merged**